CRD-DTR/Shared/R4/resources/Questionnaire-R4-PractitionerInfo.json

```diff
@@ -71,6 +75,10 @@
                 "language": "text/cql",
                 "expression": "\"BasicClinicalInfoPrepopulation\".EncounterProviderMiddleInitial"
               }
+            },
+            {
+               "url" : "http://hl7.org/fhir/StructureDefinition/minLength",
+               "valueInteger": 6
```

Comment:
Does it make sense to mandate the middle initial with minimum 6 characters?

Reply:
It also doesn't work. It doesn't show the error message if I just type 2 characters.